packages/flutter/test/material/slider_test.dart

```diff
@@ -4530,8 +4530,9 @@ void main() {
 
       // test tap
       final TestGesture gesture = await tester.startGesture(centerOfTheSlideTrack);
-      await tester.pump();
-      // has no effect as tap is disabled, remains 1.0
+      // expect interaction animation
+      expect(await tester.pumpAndSettle(), greaterThan(1));
```

Comment:
I suggest:
* When you pump 0.3 seconds, verify that you draw a circle with a certain opacity or size at the certain location
* pumpAndSettle, and verify that you you draw a circle at the certain location

Same with release.

Reply:
Thanks for the advise. I implemented it with pumpAndSettle - assuming your list is a list of alternative implementations. 

But if I misunderstood and you meant to pump for 0.3s AND pumpAndSettle, could you please elaborate, why that would be necessary?

Reply:
The 0.3s check is to ensure that the halfway animation is as expected. The pumpAndSettle check is to ensure the final state is as expected.

Reply:
Ah, that makes sense. The animation is `kRadialReactionDuration` (100ms) long. I'll pump it for 50ms then.